docs-public/snippets.js

```diff
@@ -555,7 +555,7 @@ function toModuleImport(dir) {
 
 function toSystemJsPackage(dir) {
     var key = '"' + dir.module + '"';
-    var contents = JSON.stringify({ main: './dist/npm/js/main.js', defaultExtension: 'js' });
+    var contents = JSON.stringify({ main: dir.main, defaultExtension: 'js' });
```

Comment:
@gyoshev should I leave a fallback here? I think I'll skip it as all components provide CDN bundles now.